### PR TITLE
feat: improve artisan command output readability

### PR DESCRIPTION
**Pull request by almas1992**
## 📑 Description

Optimizes the output of Artisan commands by introducing multi-color formatting.

#### Screenshots

image

image

## Summary by CodeRabbit

- **New Features**
  - Enhanced command handling with custom behavior for unrecognized commands and usage errors.
  - Improved help and version printing for CLI applications with custom formatting.
  - Suggestions for similar command names when unrecognized commands are entered.
  - Enhanced output capture for logging during command execution.

- **Bug Fixes**
  - Updated error handling logic to ensure successful exits even in error scenarios.

- **Tests**
  - Introduced unit tests for command-line interface commands, validating help and error message formatting.

- **Chores**
  - Updated application metadata for clarity and consistency.

## ✅ Checks

- [ ] Added test cases for my code

### CODE REVIEW

**Comment by hwbrzzl** [comment]
Please add the `Review Ready` tag when the PR is ready.

**Comment by almas1992** [comment]
Review Ready

File: console/cli_helper_test.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
+package console
+
+import (
+	"bytes"
+	"github.com/goravel/framework/contracts/console"
+	"github.com/goravel/framework/contracts/console/command"
+	"github.com/goravel/framework/support/color"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"testing"
```
Comment (hwbrzzl) [inline console/cli_helper_test.go:10]: Sort them:

```go
System packages

Third packages

Internal packages
```

**Review by hwbrzzl — COMMENTED** [review]
Pretty good, so beautiful 👍 Just wondering, the logic is a bit complicated. Did you write it manually or reference something?

**Comment by hwbrzzl** [comment]
And please update branch

File: console/application.go [hunk 2]
```diff
@@ -1,204 +1,207 @@
 package console
 
 import (
-	"os"
-	"strings"
+    "os"
```
[the remark this answers is not in the document]
Comment (almas1992) [reply console/application.go:4]: I am using Goland, and I have configured it to use 4 spaces for tabs.

File: console/cli_helper_test.go (hunk 1 above)
Comment (almas1992) [reply console/cli_helper_test.go:10]: Okay

**Comment by almas1992** [comment]
> And please update branch

should I update branch with `merge commit` or `rebase`?

**Comment by almas1992** [comment]
> Pretty good, so beautiful 👍 Just wondering, the logic is a bit complicated. Did you write it manually or reference something?

I manually wrote the help template for urfave/cli based on Laravel Artisan’s format and implemented the findAlternatives method in Go, which replicates the command suggestion logic from Laravel.

**Comment by hwbrzzl** [comment]
> > Pretty good, so beautiful 👍 Just wondering, the logic is a bit complicated. Did you write it manually or reference something?
> 
> I manually wrote the help template for urfave/cli based on Laravel artisan format and implemented the findAlternatives method in Go, which replicates the command suggestion logic from Laravel.

Amazing, Great work! I updated the branch, and we need to use the rebase command.

**Review by hwbrzzl — APPROVED** [review]
LGTM

**Review by hwbrzzl — APPROVED** [review]

**Pull request merged**